components/AppNav.tsx

```diff
@@ -84,31 +79,64 @@ const UnAuthLink = () => (
   </div>
 )
 
-const AppNav: React.FC<Props> = ({
-  loggedIn = false,
-  username = '',
-  firstName,
-  lastName
-}) => {
-  const initial = firstName && lastName ? firstName[0] + lastName[0] : ''
+export const AuthButtons: React.FC<AuthButtonProps> = ({
+  initial,
+  username
+}) => (
+  <div>
```

Comment:
why does the button need to be wrapped around a `div` tag?

Reply:
Use react fragments when possible: `<>....</>`

Reply:
Fixed